core/gas.h

```diff
@@ -41,12 +43,18 @@ static inline int get_he(struct gasmix mix)
 	return mix.he.permille;
 }
 
+static inline int get_n2(struct gasmix mix)
```

Comment:
This should have been added long long ago. I am sure, we will find many other places where it can replace a convoluted 1000 - o2 -he construction.